src/commonMain/kotlin/teksturepako/pakku/api/data/ConfigFile.kt

```diff
@@ -39,11 +39,8 @@ data class ConfigFile(
     /** A mutable list of client overrides packed up with the modpack. */
     @SerialName("client_overrides") private val clientOverrides: MutableList<String> = mutableListOf(),
 
-    @SerialName("mods_path") var modsPath: String? = null,
-    @SerialName("resource_packs_path") var resourcePacksPath: String? = null,
-    @SerialName("data_packs_path") var dataPacksPath: String? = null,
-    @SerialName("worlds_path") var worldsPath: String? = null,
-    @SerialName("shaders_path") var shadersPath: String? = null,
+    /**  A map of project types to their respective paths. */
+    val paths: MutableMap<ProjectType, String> = ProjectType.entries.associateWith { it.defaultPath }.toMutableMap(),
```

Comment:
`= mutableMapOf()`

Reply:
We need to show the available types. Use null as the value?

Reply:
No `mutableMapOf()` is fine. It shouldn't be serialized by default.

Reply:
How can we tell the users what are the keys available? In document?

Reply:
Yes in documentation. Config file is technically optional If you don't need to export the modpack.